torch/csrc/distributed/rpc/tensorpipe_agent.cpp

```diff
@@ -403,7 +420,15 @@ std::shared_ptr<FutureMessage> TensorPipeAgent::send(
             [this, &clientPipe](
                 const tensorpipe::Error& error, Message&& responseMessage) {
               if (error) {
-                LOG(WARNING) << "Read response error: " << error.what();
+                if (error.isOfType<tensorpipe::PipeClosedError>() &&
```

Comment:
nit: this and `ListenerClosedError` have appeared multiple times, does it make sense to extract them into one function? This is minor-minor comment, feel free to ignore.